src/store.js

```diff
@@ -4,7 +4,12 @@ export const store = {
         statusIndicator: "fine",
         foodIndicator: "100%",
         ageIndicator: 3,
-    }
+    },
+    indicators: {
+        state:  ["fine", "sick", "hungry", "dead"],
+        food: [1, 20, 40, 60, 80, 100],
+        age: [0, 1, 2, 3, 4, 5]
```

Comment:
I don't think we need pre-defined years, since the requirements say that this age is a simple integer incremented by 1 each year, like people's age. So you could start with 0 and then do simple math.

Reply:
What you could have in the config is a max year that a pet can live, like 5 in this case. could be made configurable in the future via environment variables